### ISSUE TITLE
How do build stages affect concurrent jobs?

### ISSUE THREAD

**Issue by NickHeiner** [travis-ci/travis-ci#8523]
I see in the [Plans page](https://travis-ci.com/plans) that plans are limited by number of concurrent jobs. How do build stages affect this? If I have a build stage with 2 jobs that run in parallel, do they both count against the limit?

Also, it's frustrating that I can't access the [Plans page](https://travis-ci.com/plans) when I'm logged in to Travis. It redirects me to the dashboard. I have to view the page in an Incognito window. 😦

**Comment by BanzaiMan** [comment]
Build stages affects how the jobs are queued. Each job is subject to plan capacity whether or not it is allowed to run.

I can access the plans page without being logged in. What do you see?

![10_6_17__14_45](https://user-images.githubusercontent.com/25666/31293409-1c63100a-aaa5-11e7-99d0-da45f18c5d03.png)

**Comment by NickHeiner** [comment]
> Build stages affects how the jobs are queued. Each job is subject to plan capacity whether or not it is allowed to run.

I'm sorry, I still don't fully understand this. Are you suggesting that if a job is cancelled, then it still counts against plan capacity?

Let's say I have a build with 3 jobs. If I'm on the "startup" plan, would that mean that the first two jobs could run in parallel, and then the third would run as soon as a slot opens up?

> I can access the plans page without being logged in. What do you see?

I was saying that I can't access the plans page when I **am** logged in:

![travis plans](https://user-images.githubusercontent.com/829827/31299611-48e3f5a0-aabd-11e7-99b3-399bf3f192a2.gif)

You can see here that when I navigate to `/plans`, I'm sent back to `/dashboard`.

**Comment by BanzaiMan** [comment]
> Let's say I have a build with 3 jobs. If I'm on the "startup" plan, would that mean that the first two jobs could run in parallel, and then the third would run as soon as a slot opens up?

This is true if you have no other jobs running when this build is started. This is also true when you do not use Build Stages. In other words, Build Stages _*only*_ changes when the jobs are queued, not how many at a time.

Also, I can use the plans when I am logged in.

![travis_ci_-_test_and_deploy_with_confidence](https://user-images.githubusercontent.com/25666/31300301-fabdc5fa-aac0-11e7-9466-e243379bdda4.png)

**Comment by BanzaiMan** [comment]
Also in an incognito window

![travis_ci_-_test_and_deploy_with_confidence](https://user-images.githubusercontent.com/25666/31300331-2250216c-aac1-11e7-8e26-94bc4093272b.png)

**Comment by NickHeiner** [comment]
Your screenshots are from `.com`. Mine are from `.org`. Does `/plans` work for you on `.org`?

**Comment by BanzaiMan** [comment]
`.org` does not have `/plans` page.
